src/main.tsx

```diff
@@ -2,7 +2,8 @@ import React from "react";
 import ReactDOM from "react-dom/client";
 import { Provider } from "react-redux";
 import App from "./App";
-import "./index.scss";
+import "normalize.css";
```

Comment:
did you add a reset.css?

Reply:
i read that adding of reset.css differ from normalize. and then i found a lot of links where were written that you should just put it in index.css. 
Like that: https://www.joshwcomeau.com/css/custom-css-reset/